xinference/client.py

```diff
@@ -58,10 +58,42 @@ def generate(
             Union["LlamaCppGenerateConfig", "PytorchGenerateConfig"]
         ] = None,
     ) -> Union["Completion", Iterator["CompletionChunk"]]:
+        """
+        Function to call the generate functionality of model on the server side.
+
+        Parameters
+        ----------
+        prompt : str
+            The user's message and the user's input
+        generate_config: Optional["ChatglmCppGenerateConfig"]
+            Additional Configuration for the Model text generation.
+
+        Returns
+        -------
+        Union["Completion", Iterator["CompletionChunk"]]
+            The completion chunk generated by the Model.
```

Comment:
`stream=True` -> Iterator["CompletionChunk"]
`stream=False` -> "Completion"

Reply:
resolved in the latest version